snmp/datadog_checks/snmp/data/default_profiles/_hp-compaq-health.yaml

```diff
@@ -184,12 +184,22 @@ metrics:
     symbols:
       - OID: 1.3.6.1.4.1.232.6.2.9.3.1.5
         name: cpqHeFltTolPowerSupplyStatus
-      - name: cpqHeFltTolPowerSupply
-        constant_value_one: true
       - OID: 1.3.6.1.4.1.232.6.2.9.3.1.7
         name: cpqHeFltTolPowerSupplyCapacityUsed
       - OID: 1.3.6.1.4.1.232.6.2.9.3.1.8
         name: cpqHeFltTolPowerSupplyCapacityMaximum
```

Comment:
1/ I think you can move `cpqHeFltTolPowerSupplyCapacityUsed` and `cpqHeFltTolPowerSupplyCapacityMaximum` to the new table that has `cpqHeFltTolPowerSupply` constant one metric.

Moving those metric there can be beneficial since user will be able for example to graph `CapacityUsed` grouped by `power_supply_status`.

2/ Once 1/ is done, since the old table with `cpqHeFltTolPowerSupplyStatus` won't be useful anymore (replace by constant one metric + power_supply_status tag), we can likely add a DEPRECATED comment to the table and suggest people to use the new table instead.